crates/testenv/src/utils.rs

```diff
@@ -0,0 +1,90 @@
+use bdk_chain::bitcoin;
+
+#[allow(unused_macros)]
+#[macro_export]
+macro_rules! block_id {
+    ($height:expr, $hash:literal) => {{
+        bdk_chain::BlockId {
+            height: $height,
+            hash: bitcoin::hashes::Hash::hash($hash.as_bytes()),
+        }
+    }};
+}
+
+#[allow(unused_macros)]
+#[macro_export]
+macro_rules! hash {
+    ($index:literal) => {{
+        bitcoin::hashes::Hash::hash($index.as_bytes())
+    }};
+}
+
+#[allow(unused_macros)]
+#[macro_export]
+macro_rules! local_chain {
+    [ $(($height:expr, $block_hash:expr)), * ] => {{
+        #[allow(unused_mut)]
+        bdk_chain::local_chain::LocalChain::from_blocks([$(($height, $block_hash).into()),*].into_iter().collect())
+            .expect("chain must have genesis block")
+    }};
+}
```

Comment:
Nit: I suggest renaming this expression to `hash` for consistency with the other macros.

Reply:
The previous `h!` macro has been renamed to `hash!`, so it will conflict. Since it builds a `LocalChain`, is it better to rename it `new_local_chain`?

Reply:
Oops! Just to clarify: I meant to change `block_hash` to `hash`, not rename the `local_chain` macro. Apologies for the confusion!

Reply:
done